src/content/reference/react/useId.md

```diff
@@ -163,33 +162,34 @@ input { margin: 5px; }
 
 </Sandpack>
 
-[Watch this video](https://www.youtube.com/watch?v=0dNzNcuEuOo) to see the difference in the user experience with assistive technologies.
+[Tonton video ini](https://www.youtube.com/watch?v=0dNzNcuEuOo) untuk melihat perbedaan pengalaman pengguna dengan teknologi bantu.
 
 <Pitfall>
 
-With [server rendering](/reference/react-dom/server), **`useId` requires an identical component tree on the server and the client**. If the trees you render on the server and the client don't match exactly, the generated IDs won't match.
+Dengan [rendering server](/reference/react-dom/server), **`useId` membutuhkan pohon komponen yang identik di server dan klien**. Jika pohon yang Anda *render* di server dan klien tidak sama persis, ID yang dihasilkan tidak akan cocok.
```

Comment:
```suggestion
Dengan [*server rendering*](/reference/react-dom/server), **`useId` membutuhkan pohon komponen yang identik di server dan klien**. Jika pohon yang Anda *render* di server dan klien tidak sama persis, ID yang dihasilkan tidak akan cocok.
```

As discussed in this thread: https://github.com/reactjs/id.react.dev/pull/524/files#r1181636628